src/check.ts

```diff
@@ -18,34 +31,55 @@ document.addEventListener('DOMContentLoaded', function () {
   const checkCreditcardname = document.getElementById('check__creditcardname');
   const checkCreditcardexpirationdate = document.getElementById('check__creditcardexpirationdate');
   const checkCreditcardnumber = document.getElementById('check__creditcardnumber');
+  const pspbank = document.getElementById('check__pspbank');
+  const pspbankname = document.getElementById('check__pspbankname');
+  const pspcost = document.getElementById('check__pspcost');
+  const pspchoose = document.getElementById('check__pspchoose');
+  const checkUserEmail = document.getElementById('check__useremail');
 
-  const amount = sessionStorage.getItem('amount');
-  const creditcardname = sessionStorage.getItem('creditcardname');
-  const creditcardexpirationdate = sessionStorage.getItem('creditcardexpirationdate');
-  const creditcardnumber = sessionStorage.getItem('creditcardnumber');
+  const fixedCost: number = wallet?.psp.fixedCost.amount || 0;
+  const amount: number | null = checkData?.amount.amount;
+  const totAmount: number = fixedCost && amount ? fixedCost + amount : 0;
+  const prettyAmount: number = totAmount / 100;
+  const prettyfixedCost: number = fixedCost / 100;
 
-  const creditcardcircuitValidator = CreditCard.number(creditcardnumber);
-  const creditcardcircuit =
-    creditcardcircuitValidator.card !== null ? creditcardcircuitValidator.card.type.toLowerCase() : null;
-  if (creditcardcircuit !== null) {
-    circuitCustomEl?.classList.remove('d-none');
-    circuitDefaultEl?.classList.add('d-none');
-    const circuitCustomUrl = circuitCustomTypeHref?.replace('#icons-visa', `#icons-${creditcardcircuit}`);
-
-    circuitCustomType?.setAttribute('href', circuitCustomUrl || '');
+  if (checkTotamount) {
+    // eslint-disable-next-line functional/immutable-data
```

Comment:
@vito80ba What about moving this linter directive at file level/scope?

Reply:
I believe that is better to leave this exception line-by-line to avoid problems wherever we need a useful tip by linter